src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -174,7 +178,7 @@ const FailedToAddUsersMessage: React.FC<FailedToAddUsersMessageProps> = ({
       </div>
       <div className="message-body" css={{flexDirection: 'column'}}>
         {isOpen && (
-          <MessageDetails users={users} reason={message.reason} domains={message.backends}>
+          <MessageDetails users={users} reason={message.reason} domains={users.map(user => user.domain)}>
```

Comment:
Getting the unreachable domains in the response only works in proteus, mapping the domain of the user we have in database works for both proteus and mls